### PR TITLE
add Llama Coder

### PR DESCRIPTION
**Pull request by ex3ndr**
Hey, i have created a plugin for VS Code that integrates with ollama!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -272,3 +272,4 @@ See the [API documentation](./docs/api.md) for all endpoints.
 - [Discord AI Bot](https://github.com/mekb-turtle/discord-ai-bot)
 - [Hass Ollama Conversation](https://github.com/ej52/hass-ollama-conversation)
 - [Rivet plugin](https://github.com/abrenneke/rivet-plugin-ollama)
+- [Llama Coder](https://github.com/ex3ndr/llama-coder) (Copilot alternative using ollama)
```
Comment (jmorganca) [inline README.md:275]:
```suggestion
- [Llama Coder](https://github.com/ex3ndr/llama-coder) (Copilot alternative using Ollama)
```

**Review by jmorganca — APPROVED** [review]
Looks good! Just a small typo :)

**Comment by ex3ndr** [comment]
Thanks! Sorry about this.

**Comment by jmorganca** [comment]
No problem! Thank you so much for the PR! This project looks incredibly cool I can't wait to try it 😊

**Review by jmorganca — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (09217415607) [reply README.md]: - [Llama Coder](https://github.com/ex3ndr/llama-coder) (Copilot alternative using

**Pull request merged**